src/components/PastTours/PastTourDetails.tsx

```diff
@@ -1,64 +1,72 @@
-import React, { useState } from "react"
+import React, { useState, useEffect } from "react"
 import { RouteComponentProps } from "react-router-dom"
 import "./PastTours.css"
 import circleCheck from '../../assets/purplebluecircle.png'
 import arrow from '../../assets/arrow.png'
-
+import { useAuth0 } from "@auth0/auth0-react"
 import question from "../../assets/question-sign.svg"
 import route from "../../assets/travel.svg"
 import lightbulb from "../../assets/light-bulb (1).svg"
-
-import { tourPlans } from "../../mockdata/tourPlan"
+import { getPlan, cleanInputStrings } from "../../util.js"
 
 interface tourPlan {
```

Comment:
I think convention may be to have this be CamelCase but looks like it's not affecting functionality anyway.